lib/file-helpers.sh

```diff
@@ -24,6 +24,7 @@ export bashmatic__temp_file_pattern=".bashmatic.${bashmatic__hostname}.${USER}."
 
 .file.remote_size() {
   local url="$1"
+  ## :TBD: Wouldn't `cut -d':' -f2` be cheaper than `awk`?
```

Comment:
Quite possibly, although I've been more bit by `cut` than `awk` :)

Reply:
It's quite a bit faster -- awk's  great if you have more logic needed, but this is pretty cut'n'dried